src/main/java/com/example/helloworld/HelloWorldController.java

```diff
@@ -0,0 +1,15 @@
+package com.example.helloworld;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class HelloWorldController {
+
+    @GetMapping("/hello")
+    public String hello() {
+        String message="hello world";
```

Comment:
プロジェクトごとに設定するのが面倒ですがフォーマッタの設定をお願いします！
```suggestion
        String message = "hello world";
```